models/settings_helpers.go

```diff
@@ -135,6 +138,9 @@ func UpdateSettings(q reform.DBTX, params *ChangeSettingsParams) (*Settings, err
 	if params.EnableSTT {
 		settings.SaaS.STTEnabled = true
 	}
+	if params.EnableDBaaS {
+		settings.DBaaS.DBaaSEnabled = true
```

Comment:
⚠️ Until we have `params.DisableDBaaS` and API for that, it should be `settings.DBaaS.DBaaSEnabled = params.EnableDBaaS`. Otherwise, there is not way to disable DBaaS without wiping out everything.

Reply:
Good point